CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing
+
+While in preview we maintain only a `master` branch for releases and `dev` branch for development.
+
+After release we will maintain a branch for each major version to backport important fixes.
+
+Please submit pull requests to `dev` to update the latest version or to a version branch for backports.
```

Comment:
Are there any other specific developer requirements, such as using `godep` for handling dependencies to be able to run tests, requirements for writing tests and integrating them, automated checks that need to pass, etc?

Reply:
Also make sure to mention the minimum required version of Go for development. This might be different than the minimum version to use the libraries, if I remember right.

Reply:
thanks for raising these @sptramer! @marstr we should discuss. i'll work on this, but would it be okay to land the basic page as proposed now and add these later (e.g. by 14.0.0)? in particular i'm not sure we have a well-documented test process yet.

Reply:
added this. for now we support 1.8 and 1.9, we're working on 1.10 (#964) and considering 1.7 (#965).